src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -172,6 +172,11 @@ public VisitArgumentResult(VisitResult visitResult, Optional<LocalState> stateFo
         /// </summary>
         private PooledDictionary<BoundExpression, TypeWithState> _methodGroupReceiverMapOpt;
 
+        /// <summary>
+        /// State of awaited expressions on the stack, for substitution in placeholders within GetAwaiter calls.
+        /// </summary>
+        private PooledDictionary<BoundAwaitableValuePlaceholder, (BoundExpression, VisitResult)> _awaitablePlaceholdersOpt;
```

Comment:
>(BoundExpression, VisitResult) [](start = 65, length = 30)

Consider specifying names, so that it would be clear what is the meaning of information that we are storing in this map.